pages/about.js

```diff
@@ -1,24 +1,47 @@
-import Link from 'next/link'
+import Link from "next/link";
 import styles from "@/styles/Home.module.css";
-export default function about() {
-    return (
+"use client";
+import { useUser } from "@clerk/nextjs";
```

Comment:
Due to this deployment is failing please check this.

Reply:
This is a use to get if user is loggen in or not, I can't say how it can affect code in any way? @ank1traj